site/content/user-guide/amqp-adapter.md

```diff
@@ -145,7 +149,7 @@ Publish some JSON data for device `4711` using a client certificate for authenti
   * Rejected: Adapter rejects the message due to:
         * (`hono:bad-request`): A bad client request (e.g invalid content-type).
         * (`amqp:unauthorized-access`): The adapter is disabled for tenant.
-        * (`amqp:precondition-failed`): Request not fulfilling certain requirements.
+        * (`amqp:precondition-failed`): Request not fulfilling certain requirements e.g the message limit for the given tenant is exceeded etc.
```

Comment:
The AMQP spec defines a nice `amqp:resource-limit-exceeded` error which seems quite appropriate for this case, don't you think? The Dispatch Router also uses it cases where a client exceeds its link limit ...

Reply:
`amqp:resource-limit-exceeded` sounds more appropriate than the `amqp:precondition-failed`. I will create a separate PR for that.

Reply:
@sophokles73 I have raised PR #1265 for the above.

Reply:
Also the AMQP adapter user guide has been updated with the error `amqp:resource-limit-exceeded`.